pages/introduction.md

```diff
@@ -13,7 +13,7 @@ However, when Federal agencies are unable to identify an existing Federal or COT
+While the benefits of enhanced Federal code reuse are significant, additional benefits can accrue when code is also made available to the public as Open Source Software (OSS). Making code available with an OSS license can enable continual improvement of Federal code projects when a broader community of users implements the code for its own purposes and publishes bug reports, fixes, and improvements. A number of private sector companies have already shifted some of their software development projects to an open source model,<sup id="fnr8"><a href="#fn8">8</a></sup> in which the source code of the software is made broadly available to the public for inspection, improvement, and reuse. In fact, several Federal agencies and component organizations also have already begun publishing custom-developed code under open source licenses or in the public domain, as discussed further below. Moreover, the Administration made a commitment, as part of its Second Open Government National Action Plan,<sup id="fnr9"><a href="#fn9">9</a></sup> to develop an Open Source Software policy that, together with the U.S. Digital Services Playbook,<sup id="fnr10"><a href="#fn10">10</a></sup> will support improved access to custom code developed for the Federal Government. This policy fulfills that commitment in an effort to improve U.S. Government software development and make the Government more open, transparent, and accessible to the public. Just as the Administration's Open Data Policy<sup id="fnr11"><a href="#fn11">11</a></sup> contributed to the creation of valuable and successful private businesses and services based upon open data released by the Government,<sup id="fnr12"><a href="#fn12">12</a></sup> improving access to taxpayer-funded source code can help facilitate similar results predicated on OSS.
```

Comment:
The diff here doesn't display very well, but the suggestion here is changing

> users implements the code for its own purposes and publishes bugs and improvements

to

> users implements the code for its own purposes and publishes bug **reports, fixes,** and improvements

"Publishing bugs" isn't something that software developers would say. Instead, contributors might let the project maintainers know that a certain bug exists ("reporting" it), and/or submit a fix for that problem. This should be an innocuous improvement to the language of the policy.